CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 ## Changelog
 
+### v1.0.8
+
+**New CLI options available, mainly ability to override hooks worker timeouts**
+- [#382](https://github.com/apiaryio/dredd/pull/382) Pass in options to hook worker (@xvalentino)
+- [#414](https://github.com/apiaryio/dredd/pull/414) Pass in options to hook worker (continued) (@honzajavorek)
```

Comment:
Sure, see http://dredd.readthedocs.org/en/latest/usage/#cli-interface. I know the docs aren't ideal (copy-pasted output of `--help`), I'd like to change them and possibly auto-generate them by a script before release or sth similar (tracking issue would be probably #377).